docs/content/Header.mdx

```diff
@@ -54,25 +54,25 @@ All items directly under the Header component should be a `Header.Item` componen
 </Header>
 ```
 
-## Component props
+## Props
 
-### Header
+### `Header`
 
-| Name | Type              | Default | Description                          |
-| :--- | :---------------- | :-----: | :----------------------------------- |
-| sx   | SystemStyleObject |   {}    | Style to be applied to the component |
+<PropsTable>
+  <PropsTableSxRow />
+</PropsTable>
 
-### Header.Item
+### `Header.Item`
 
-| Name | Type              | Default | Description                                |
-| :--- | :---------------- | :-----: | :----------------------------------------- |
-| full | Boolean           |         | stretches item to fill the available space |
-| sx   | SystemStyleObject |   {}    | Style to be applied to the component       |
+<PropsTable>
+  <PropsTableRow name="full" type="string" description="Stretches item to fill the available space" />
+  <PropsTableSxRow />
+</PropsTable>
 
-### Header.Link
+### `Header.Link`
```

Comment:
```suggestion
### Header.Link
```

Reply:
Swapped it back, but it feels pretty ugly as the other headings include object syntax.